### DISCUSSION TITLE
ohmyfetch withCredentials

### DISCUSSION THREAD

**Discussion by k0mar12** [nuxt/nuxt#17528] [Questions] [answered]
Hey there.

Is it possible to set option `withCredentials` using `$fetch`?

**Comment by danielroe** [comment, marked as answer]
Yes. Via `credentials: 'include'` - see https://developer.mozilla.org/en-US/docs/Web/API/fetch for docs.

**Reply by k0mar12** [reply]
Yes, sure, thank you :)

**Reply by Donybro** [reply]
Hi !
i am using nuxt 3 , and trying to set withCredentials:true 
this is my code
`import { $fetch } from 'ohmyfetch'
const apiFetch = $fetch.create({baseURL:baseURLDev,credentials:'include'})`

i did like this but it didnt work.
there is no parameter in $fetch.create() called credentials
Error message
`Argument type {baseURL: string, credentials: string} is not assignable to parameter type FetchOptions   Object literal may only specify known properties, but the following are extra: 'credentials'`

**Reply by cdwmhcc** [reply]
same
